src/time_tagger/builder.py

```diff
@@ -0,0 +1,20 @@
+from itertools import combinations
+import TimeTagger as TT
+
+
+class TimeTaggerBuilder:
+    def __init__(self) -> None:
+        pass
+
+    def build_coincidence_virtual_channel(
```

Comment:
does this need to be in a class?

Reply:
It is not so necessary for now, but I believe that we will have more builders here soon.

Reply:
actually, this can be moved to the proposed class Tagger